### PR TITLE
feat: #58-maps-project

### LINKED ISSUE #58: Check And Update Before Receiving Modal
### Explanation:
Change of informative modal styles, change font size, alert icon, button color, and transparency background.
Graphic resources are available in the attached figma links below.

### SCREENSHOTS
image

### CHECKLIST

- [ ] Change alert icon.
- [ ] Change of font and font sizes type: Poppins bold and regular according to design text color `#3B3B3C` 
- [ ] Transparency wallpaper for popup `#0000FF` opacity 17%.
- [ ] Change color and letter in button `#005D9D`

### REFERENCES
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=420%3A148
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=522%3A28

### PR DESCRIPTION
**Pull request by ygamboalh**
### Description

1. Change alert icon
2. Change of font and font sizes according to design
3. Transparency of the wallpaper in popup.
4. Change color and letter in button

Font family won't work. It will work when someone who has the task, add the font named Poppins.

fixes #58  (issue)

### How to test it
1. Go to /check 

### Checklist:

- [x]  Change alert icon.
- [x]  Change of font and font sizes type: Poppins bold and regular according to design text color #3B3B3C
- [x]  Transparency wallpaper for popup #0000FF opacity 17%.
- [x]  Change color and letter in button #005D9D

### CODE REVIEW

**Review by e-azocar — APPROVED** [review]
### It's ok, but just a comment, the design has `Poppins` as a `font family`
image

**Comment by ygamboalh** [comment]
> ### It's ok, but just a comment, the design has `Poppins` as a `font family`
> image

Yes, but I was told to leave it that way. It's someone else task to include that font family. If I'd do it then that task would be done twice. What I want to say is that when the other PR is accepted, the font will be visible because I already specified the font family.

**Comment by e-azocar** [comment]
@ygamboalh Ok, I approved the PR

**Comment by angelik0828** [comment]
In addition to fonts, you need to check font size and icon size. I place the image that is in the design and how it should be:


image
image

This is how it looks on mobile the icon is too big

image

**Comment by ygamboalh** [comment]
Review.
Trying to get a close result to the design suggested, font size for text was set to: 
--16px instead of 14px. (p)
--16px instead of 14px. (ul)
--24px instead of 20px.(h1)

Font family was set to Poppins and Work sans according to figma design.
Icon size was changed according to the figma design.

Please make suggestions if there's something missing.

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,8 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Work Sans' rel='stylesheet'>
```
Comment (heliomar-pena) [inline public/index.html]: Work Sans? @angelik0828 please confirm

File: src/components/Form/Form.styled.jsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const FormContainer = styled.form`
   display: flex;
   align-items: center;
   flex-direction: column;
+
```
Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]: please remove extra space

File: src/layouts/Splitted.styled.jsx [hunk 3]
```diff
@@ -5,7 +5,8 @@ export const Container = styled.div`
   flex-direction: column;
   min-height: 100vh;
   width: 100vw;
-  background: var(--white);
+  background-color: #0000FF;
```
Comment (heliomar-pena) [inline src/layouts/Splitted.styled.jsx:8]: I think later can be added an issue to convert hexadecimal codes in variables and replace it

File: src/pages/Home/Home.styled.jsx [hunk 4]
```diff
@@ -141,5 +142,6 @@ export const Button = styled(Link)`
     letter-spacing: 2px;
     transform: none;
     max-width: 378px;
+
```
Comment (heliomar-pena) [inline src/pages/Home/Home.styled.jsx]: Please remove extra spaces

File: src/pages/Home/Home.styled.jsx [hunk 5]
```diff
@@ -18,6 +18,7 @@ export const Header = styled.div`
   @media (min-width: 768px) {
     flex-wrap: nowrap;
     flex-direction: row;
+
```
Comment (heliomar-pena) [inline src/pages/Home/Home.styled.jsx]: Please remove extra spaces

File: src/styles/text.js [hunk 6]
```diff
@@ -25,6 +25,10 @@ const text = {
     font-size: 1.5rem;
     line-height: 2rem;
   `,
+  '1xl': css`
+  font-size: 0.875rem;
+  line-height: 1.3;
+`,
```
Comment (heliomar-pena) [inline src/styles/text.js]: Please fix the nesting

**Review by heliomar-pena — APPROVED** [review]
Good work! I have left you some comments to fix and later the code can be merged

File: src/components/Form/Form.styled.jsx [hunk 7]
```diff
@@ -1,19 +1,18 @@
...
-  width: 100%;
-  height: 100%;
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+  width:100%;
+  height:100%;
+  display:flex;
+  align-items:center;
+  flex-direction:column;
```
Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]:
```suggestion
  width: 100%;
  height: 100%;
  display: flex;
  align-items: center;
  flex-direction: column;
```

File: src/components/Form/Form.styled.jsx [hunk 8]
```diff
@@ -1,19 +1,18 @@
...
-  width: 280px;
-  height: 40px;
-  border-radius: 2rem;
-  margin: 10px;
-  margin-top: 60px;
+  width:280px;
+  height:40px;
+  border-radius:2rem;
+  margin:10px;
+  margin-top:60px;
```
Comment (heliomar-pena) [inline src/components/Form/Form.styled.jsx]:
```suggestion
  width: 280px;
  height: 40px;
  border-radius: 2rem;
  margin: 10px;
  margin-top: 60px;
```

File: src/layouts/Splitted.styled.jsx [hunk 9]
```diff
@@ -35,12 +35,11 @@ export const Header = styled.div`
     flex: none;
   }
 `
-
 export const TicketImage = styled.img`
   width: 140px;
   align-self: flex-start;
   margin: 2.5rem 0 -13px 15px;
-
+
```
Comment (heliomar-pena) [inline src/layouts/Splitted.styled.jsx]:
```suggestion

```

File: src/layouts/Splitted.styled.jsx [hunk 10]
```diff
@@ -49,12 +48,11 @@ export const TicketImage = styled.img`
     margin-left: -50px;
   }
 `
-
 export const ConfirmImage = styled.img`
   width: 140px;
   align-self: flex-start;
   margin: 0 0 -13px 15px;
-
+
```
Comment (heliomar-pena) [inline src/layouts/Splitted.styled.jsx]:
```suggestion

```

File: src/pages/Home/Home.styled.jsx [hunk 11]
```diff
@@ -17,7 +17,7 @@ export const Header = styled.div`
 
   @media (min-width: 768px) {
     flex-wrap: nowrap;
-    flex-direction: row;
+    flex-direction: row;
```
Comment (heliomar-pena) [inline src/pages/Home/Home.styled.jsx]:
```suggestion
    flex-direction: row;
```

**Review by heliomar-pena — APPROVED** [review]
Very good work! I let you some code-styles fixes. I'm waiting @angelik0828 confirm if "Work sans" font is used in some place of the project or if it can be removed from the code

**Comment by ygamboalh** [comment]
Please review.

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Pull request merged**